test/development/acceptance-app/rsc-build-errors.test.ts

```diff
@@ -350,6 +350,34 @@ describe('Error overlay - RSC build errors', () => {
     await cleanup()
   })
 
+  it('should display error message when module resolve error occurred', async () => {
```

Comment:
Can we also add assertions what happens if we patch the file after the error occurred? I remember not being able to remove certain errors until I restarted the dev server and we  should encode that behavior in tests.

Reply:
added